gcb/patch-announce/cloudpobuild.yaml

```diff
@@ -0,0 +1,65 @@
+timeout: 1200s
```

Comment:
...what's the "po" in "cloudpobuild" stand for?

Reply:
Stands for a strange typo :)